README.md

```diff
@@ -1,19 +1,34 @@
-# .NET Core Samples
+# .NET Samples
 
-This folder contains all the sample code that is part of any topic under
-the .NET Core documentation. There are several different projects that
+This repo contains all the sample code that is part of any topic under
+the .NET documentation. There are several different projects that
 are organized in sub-folders. These sub-folders are organized similar
-to the organization of the docs for .NET Core.
+to the organization of the docs for .NET.
 
 Some of the articles will have more than one sample associated with them. 
 
 The readme.md file for each sample will refer to the article so that
 you can read more about the concepts covered in each sample.
 
+There are two classes of code in this repository:
+
+- **Samples** represent programs that demonstrate apoplication or library scenarios. These samples are typically larger, and often use more than one technology, feature, or toolkit.
+- **Snippets** represent small focused examples that demonstrate one feature or syntax. These should be no more than a single screen of code.
+
+Both samples and snippets should be buildable projects. Those projects should build and
+run on the widest set of platforms possible for the given sample. In practice, that means building .NET Core based console applications where possible. Samples that are specific to the web or a UI framework should add those tools as needed. Examples include web applications, mobile apps, WPF or WinForms apps and so on.
+
+We are working toward having a CI system in place for all code. When you make any updates to samples, make sure each update is part of a buildable
+project. Ideally, add tests for correctness on samples as well.
+
+## Snippets
+
+Snippets are extracted from small programs that include the snippet. Snippets are all located in the top level **/snippets** folder. While snippets are small blcoks of code, we want to move toward snippets that are part of buildable sample projects.
+
 ## Building a sample
 
-You build the samples using the .NET Core CLI. You can download the CLI from
-[the .NET Core home page](http://microsoft.com/net/core). Then, execute
+You build any .NET core sample using the .NET Core CLI. The .NET Core CLI is installed with
+[the .NET Core SDK](https://www.microsoft.com/net/download). Then, execute
 these commands from the CLI in the directory of any sample:
```

Comment:
In this paragraph three sentences: active - passive - active voice and a lot of *.NET Core*. Should we combine the first two sentences:
*You build any .NET Core sample using the .NET Core CLI, which can be installed with [the .NET Core SDK]....*
Also typo in the first sentence: *.NET core*

Reply:
wow. Super quick feedback.  🏎